src/App.js

```diff
@@ -0,0 +1,24 @@
+import React, { Component } from 'react'
+import ReactDOM from 'react-dom'
+import Login from './Login'
+import Menu from './Menu'
+
+export default class App extends Component {
+    render() {
+        const menuItems = [
+            {href: " / ", title: "Заглушка1"},
```

Comment:
```suggestion
            {href: "/", title: "Заглушка1"},
```
лишние пробелы не добавляем ;), но я понимаю, что ты хотел это ради стилизации, но в данном случае это может быть не просто стилизация кода, а прям пробелы в url